foundation-ui/utils/ReactUtil.js

```diff
@@ -15,6 +15,10 @@ const ReactUil = {
       return null;
     }
 
+    if (Array.isArray(elements) && elements.length === 1 && !alwaysWrap) {
```

Comment:
Please add check to verify that the element is a valid react element to avoid errors.